style-guide/event.html

```diff
@@ -0,0 +1,232 @@
+    						<figcaption class="wp-caption-text">
+    							Bitter Rivalry Ale uses only Washington-produced ingredients. (Photo courtesy of Joseph’s Grainery)
+    						</figcaption>
+    					</figure>
+            </div>
+            <section class="row side-right card-about">
+              <div class="column one">
+                <h2>About the event</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                <ul class="card-related-links">
+                  <li>
+                    <a href="#" class="related-link">Department of Something</a>
+                  </li>
+                  <li>
+                    <a href="#" class="related-link">Named College of Study</a>
+                  </li>
+                </ul>
+              </div>
+              <div class="column two">
+                <h2>Contact</h2>
+                <p class="card-contact-details">
+                  <span class="card-contact-name">Firstand Lastname</span>
+                  <span class="card-contact-title">Is this needed</span>
+                  <a href="mailto:#" class="card-contact-email">email@wsu.edu</a>
+                  <a href="tel:1-509-335-8888" class="card-contact-tel">509 335-8888</a>
+                </p>
+              </div>
+            </section>
+						<section class="row side-right card-footer">
+							<div class="column one card-taxonomy">
+								<div class="card-categories">
+									<h2>Catagorized</h2>
+									<a class="card-category" href="#">health</a>, <a href="#" class="card-category">arts</a>, <a href="#" class="card-category">performing arts</a>
```

Comment:
Do we need the specificity of `card-category`, `card-tag`, etc... when the parent has the taxonomy embedded in its class (i.e. `card-categories`)?